tools/integration_tests/util/setup/setup.go

```diff
@@ -211,6 +211,14 @@ func UnMountAndThrowErrorInFailure(flags []string, successCode int) {
 	if successCode != 0 {
 		f := strings.Join(flags, " ")
 		log.Print("Test Fails on " + f)
+
+		logData, err := os.ReadFile(LogFile())
```

Comment:
Log file will have logs for all tests. What do you mean by printing logs of failed tests?

Reply:
Also why are we printing in console and not uploading to kokoro?

Reply:
> Also why are we printing in console and not uploading to kokoro?

We are printing in kokoro console right?

Reply:
> Log file will have logs for all tests. What do you mean by printing logs of failed tests?

Log file will have log of that mount on which test fails.

Reply:
We are printing in kokoro console right?
It will print logs on test fusion.

>> why are we not uploading to kokoro. wont the test fusion get bloated up logs considering there are lot of tests under each mount?

Reply:
You mean to upload the logs on bucket ?

Reply:
sorry, yes bucket. I meant use as kokoro artifact

Reply:
fixed